Makefile

```diff
@@ -69,3 +70,19 @@ flake: ## 🧹 Run flake8
 updatekv: ## 🔄 Update keyvault secrets
 	$(call target_title, "updating keyvault secrets")
 	python3 env_to_keyvault.py
+include .env
+export
+
+deploy:
```

Comment:
The other makefile targets have comments/documentation to support "make help"
Suggest you add similar